tests/integration.tags

```diff
@@ -14,4 +14,3 @@
 @rekey_v1
 @send
 @send.keyregtxn
-@simulate
```

Comment:
@ahangsu I decided to take this tag out until the tests are fixed

Reply:
sure, will fix soon!